stripe/stripe-android
# Native app2app support

# Summary

Adds app2app support. If you have the Chase app installed, you'll get redirected to the app to connect accounts instead of having to input your credentials.

- adds `link-native-accounts` deeplink to return from bank app back to the SDK.
- redirects to native activity upon ^ to continue the AuthFlow.

# How to test

- Ensure you've the Chase app installed on Android
- Ping @carlosmuvi-stripe to get a test build invite!

# Testing
